agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -979,7 +1221,7 @@ def auto_semantic_similarity(
 EVALUATOR_FUNCTIONS = {
     "auto_exact_match": auto_exact_match,
     "auto_regex_test": auto_regex_test,
-    "field_match_test": field_match_test,
```

Comment:
Will this break existing evaluators, in production?
If so, we might need to keep both keys here.

Reply:
Thank you for catching that.